controllers/consoleplugin_resource_reconciler.go

```diff
@@ -364,6 +364,13 @@ func (r *ConsolePluginResourceReconciler) setDesiredConsolePluginDeployment(
 			Protocol:      corev1.ProtocolTCP,
 		},
 	}
+
+	varTrue := true
+	consolePluginContainer.SecurityContext = &corev1.SecurityContext{
+		AllowPrivilegeEscalation: &varTrue,
```

Comment:
There is another option when boolean var pointers are required. By using [this](https://pkg.go.dev/k8s.io/utils/pointer#Bool) function, we could rewrite this part like this:

```go
import "k8s.io/utils/pointer"
...

AllowPrivilegeEscalation: pointer.Bool(true),
```

We don't have to use it in this case, it's already clear and readable as it is. We can just keep it in mind, as the `pointer` pkg could lead to increased consistency and less cognitive load in such cases.

Reply:
Good point. I don't like the other syntax.
I've updated the code to use `pointer`.